### ISSUE TITLE
Data field of type __markedentry from JabRef

### ISSUE THREAD

**Issue by adamliter** [ge-ne/bibtool#8]
Hi, it seems there is a problem with BibTool processing entries that have been marked via JabRef.
If you mark an entry in the JabRef GUI program, it adds a data field of the type `__markedentry` to the entry. If you try to process this file with BibTool, it returns something like:

```
*** BibTool WARNING:  (line 5 in ./unified-test.bib): 48 non-space characters ignored.

  __markedentry            = {[adamliter:3]}
___^
*** BibTool ERROR:  (line 23 in ./unified-test.bib): Symbol does not start with a letter
*** BibTool WARNING: Skiping to next '@'
```
and it will not print the offending entry in the output.
Is there a fix for this? I'm trying to use BibTool to sanitize a `.bib` file maintained with JabRef.

**Comment by ge-ne** [comment]
Hm, I thought that I have seen all strange border cases...
I will fix this later today.

**Comment by adamliter** [comment]
Sounds good. Thanks! And thanks for BibTool! It's a great tool. `:)`

**Comment by ge-ne** [comment]
Fixed in 2.60, which has been deployed on github, gerd-neugebauer.de and submitted to CTAN

**Comment by adamliter** [comment]
Thanks!
